### PR TITLE
Created and added config.yml files

### PR DESCRIPTION
**Pull request by jerrycowell96**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: GitHub Community Support
+    url: https://github.com/orgs/community/discussions
```
Comment (edgarriba) [inline .github/ISSUE_TEMPLATE/config.yml:4]: I’d link here to discourse ? @Hackerman342

Comment (Hackerman342) [reply .github/ISSUE_TEMPLATE/config.yml:4]: Yes!

Comment (jerrycowell96) [reply .github/ISSUE_TEMPLATE/config.yml]: it has been added

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: farm-ng Community Support
+    url: https://discourse.farm-ng.com/
+    about: Please ask and answer questions here.
+  - name: GitHub Security Bug Bounty
+    url: https://bounty.github.com/
+    about: Please report security vulnerabilities here.
```
Comment (Hackerman342) [inline .github/ISSUE_TEMPLATE/config.yml:8]:
```suggestion
  - name: Open a bug report
    url: https://github.com/farm-ng/farm-ng-amiga/issues/new/choose
    about: Please open bug reports in the `farm-ng-amiga` repository.
```

**Review by Hackerman342 — APPROVED** [review]

**Pull request merged**